### PR TITLE
Add Analytics

### PR DESCRIPTION
**Pull request by dhruhin**
Set up event tracking on the website using Firebase Analytics.
I use JQuery to listen to buttons being clicked, if there's a more effecient way, lmk.
Start off analytics by tracking which designs people are clicking on learn more about.
If there are other extras, we need to track, add it to the doc here with extras: https://www.notion.so/c773bbe84eb2415085370204cb297b09?v=cd90a745a0e542598518916dc34d13ec
image

### CODE REVIEW

**Review by jpbesgen — CHANGES_REQUESTED** [review]
Thanks Dhruhin, this looks great!

File: js/components/DesignCardComponent.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ class DesignCardComponent extends Component {
         if(design.attachments != null && design.attachments.length > 0) {
             downloads = `Attachments: `
             design.attachments.forEach((attachment) => {
-                downloads += `<a href="${attachment.url}" target="_blank" download> ${attachment.name}</a>`;
```
Comment (amadrzyk) [inline js/components/DesignCardComponent.js:126]: Super minor, but do you mind adding `rel="noopener noreferrer"` to the links? Recently found out about this vulnerability 😬 

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

**Review by amadrzyk — APPROVED** [review]
looks great to me, thanks Dhru! 💯

File: js/components/DesignCardComponent.js (hunk 1 above)
Comment (dhruhin) [reply js/components/DesignCardComponent.js:126]: Of course, added this change, landing it

**Pull request merged**